hibernate-core/src/test/java/org/hibernate/orm/test/id/uuid/interpretation/UUIDBasedIdInterpretationTest.java

```diff
@@ -78,6 +79,7 @@ private void checkUuidTypeUsed(StandardServiceRegistry ssr, Class<? extends Type
 
 	@Test
 	@TestForIssue( jiraKey = "HHH-10564")
+	@RequiresDialect( MySQL5Dialect.class )
```

Comment:
In Hibernate 6 we don't use the version specific dialect sub-classes anymore but configure the dialect during construction instead, so please use:

```suggestion
	@RequiresDialect( MySQLDialect.class )
```